### PR TITLE
Add localization for UI strings in French

### PR DESCRIPTION
**Pull request by stepankuzmin**
Hey 👋

This PR adds translations for UI strings we use in GL JS into French. We translated the UI strings automatically using the Google Translate API, and we need to verify that the translation is correct.

To help us verify the translations, you can open the `Files changed` tab on this page and compare the translations with the original file [`src/ui/default_locale.js`](https://github.com/mapbox/mapbox-gl-js/blob/main/src/ui/default_locale.js).

If the translation looks good, you can approve it using the `Review changes` button in the `Files changed` tab and selecting the `Approve` radio button when submitting the review.

To improve the machine translation, you can click the plus sign next to the line you want to enhance and select the suggestion button on the toolbar.

After making the suggestions, you can mark the review as finished using the `Review changes` button and selecting the `Request changes` radio button.

Thanks for helping 🙌

### CODE REVIEW

File: src/ui/locales/fr.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "GeolocateControl.LocationNotAvailable": "Lieu non disponible",
+    "LogoControl.Title": "Logo MapBox",
+    "Map.Title": "Carte",
+    "NavigationControl.ResetBearing": "Réinitialiser le relèvement vers le nord",
+    "NavigationControl.ZoomIn": "Agrandir",
+    "NavigationControl.ZoomOut": "Dézoomer",
+    "ScaleControl.Feet": "pi",
+    "ScaleControl.Meters": "m",
+    "ScaleControl.Kilometers": "kilomètres",
+    "ScaleControl.Miles": "moi",
```
Comment (karimnaaji) [inline src/ui/locales/fr.js:18]:
```suggestion
    "ScaleControl.Miles": "milles"
```

Comment (AgentSmith0) [inline src/ui/locales/fr.js:18]: moi and pi? It should be ft and mi, right? In other languages there are also problems with wrong translations.

**Review by AgentSmith0 — CHANGES_REQUESTED** [review]

File: src/ui/locales/fr.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "FullscreenControl.Enter": "Passer en mode plein écran",
+    "FullscreenControl.Exit": "Quitter le mode plein écran",
+    "GeolocateControl.FindMyLocation": "Trouver ma position",
+    "GeolocateControl.LocationNotAvailable": "Lieu non disponible",
+    "LogoControl.Title": "Logo MapBox",
+    "Map.Title": "Carte",
+    "NavigationControl.ResetBearing": "Réinitialiser le relèvement vers le nord",
+    "NavigationControl.ZoomIn": "Agrandir",
+    "NavigationControl.ZoomOut": "Dézoomer",
+    "ScaleControl.Feet": "pi",
```
Comment (karimnaaji) [inline src/ui/locales/fr.js:15]: I think `pi` is canadian only, 
```suggestion
    "ScaleControl.Feet": "pieds",
```

File: src/ui/locales/fr.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+const locale = {
+    "AttributionControl.ToggleAttribution": "Basculer l'attribution",
+    "AttributionControl.MapFeedback": "Commentaires sur la carte",
+    "FullscreenControl.Enter": "Passer en mode plein écran",
+    "FullscreenControl.Exit": "Quitter le mode plein écran",
+    "GeolocateControl.FindMyLocation": "Trouver ma position",
+    "GeolocateControl.LocationNotAvailable": "Lieu non disponible",
+    "LogoControl.Title": "Logo MapBox",
+    "Map.Title": "Carte",
+    "NavigationControl.ResetBearing": "Réinitialiser le relèvement vers le nord",
```
Comment (karimnaaji) [inline src/ui/locales/fr.js:12]:
```suggestion
    "NavigationControl.ResetBearing": "Réinitialiser l'orientation vers le nord",
```

File: src/ui/locales/fr.js (hunk 1 above)
Comment (banzzai) [reply src/ui/locales/fr.js:18]: Probabaly ft and mi or mile. I don't know of abbreviated french version of those units. pi would be literal. Why isn't kilometers represented by "km" though?

File: src/ui/locales/fr.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    "NavigationControl.ZoomIn": "Agrandir",
+    "NavigationControl.ZoomOut": "Dézoomer",
+    "ScaleControl.Feet": "pi",
+    "ScaleControl.Meters": "m",
+    "ScaleControl.Kilometers": "kilomètres",
```
Comment (karimnaaji) [inline src/ui/locales/fr.js:17]:
```suggestion
    "ScaleControl.Kilometers": "km",
```

File: src/ui/locales/fr.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Basculer l'attribution",
+    "AttributionControl.MapFeedback": "Commentaires sur la carte",
```
Comment (karimnaaji) [inline src/ui/locales/fr.js:5]:
```suggestion
    "AttributionControl.MapFeedback": "Donner son avis sur la carte",
```

File: src/ui/locales/fr.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Basculer l'attribution",
```
Comment (karimnaaji) [inline src/ui/locales/fr.js:4]:
```suggestion
    "AttributionControl.ToggleAttribution": "Afficher l'attribution",
```

File: src/ui/locales/fr.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+    "AttributionControl.MapFeedback": "Commentaires sur la carte",
+    "FullscreenControl.Enter": "Passer en mode plein écran",
+    "FullscreenControl.Exit": "Quitter le mode plein écran",
+    "GeolocateControl.FindMyLocation": "Trouver ma position",
+    "GeolocateControl.LocationNotAvailable": "Lieu non disponible",
```
Comment (karimnaaji) [inline src/ui/locales/fr.js:9]:
```suggestion
    "GeolocateControl.LocationNotAvailable": "Localisation non disponible",
```

File: src/ui/locales/fr.js [hunk 8]
```diff
@@ -0,0 +1,25 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Basculer l'attribution",
+    "AttributionControl.MapFeedback": "Commentaires sur la carte",
+    "FullscreenControl.Enter": "Passer en mode plein écran",
+    "FullscreenControl.Exit": "Quitter le mode plein écran",
+    "GeolocateControl.FindMyLocation": "Trouver ma position",
```
Comment (karimnaaji) [inline src/ui/locales/fr.js:8]:
```suggestion
    "GeolocateControl.FindMyLocation": "Afficher ma position",
```

File: src/ui/locales/fr.js [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+    "GeolocateControl.LocationNotAvailable": "Lieu non disponible",
+    "LogoControl.Title": "Logo MapBox",
```
Comment (karimnaaji) [inline src/ui/locales/fr.js:10]: "LogoControl.Title": "Mapbox logo",

File: src/ui/locales/fr.js (hunk 1 above)
Comment (stepankuzmin) [reply src/ui/locales/fr.js:18]: Yep, it's the Google Translate mistakes

**Pull request merged**